### PR TITLE
bindinfo,planner,statistics: add `OWNERS` files in top module folders

### LINKED ISSUE #46967: prealloc wrong size for NewTopN
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

image

the size of TopN is the same as protoTopN

### LINKED ISSUE #46969: add `OWNERS` file for `pkg/bindinfo` folder

### PR DESCRIPTION
**Pull request by wuhuizuo**
### What problem does this PR solve?

Issue Number: close #46969 #46970 #46971

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [x] None
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [x] None
- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: OWNERS_ALIASES
Comment (hawkingrei) [inline OWNERS_ALIASES:28]: It would be better if you directly write it as "planner" and then add all the colleagues within the planner group.

Comment (wuhuizuo) [reply OWNERS_ALIASES:28]: Early I have the same idea: assign by nature core module names. But consider the refactor plan of folder structure raised by @bb7133 and members maybe changed in each module.

**Review by hi-rustin — CHANGES_REQUESTED** [review]
Do I have permission to review the code of these modules? If I don't have it, why take away my permission?

**Comment by wuhuizuo** [comment]
> What's the future process of changing the three owner files?

- Create pull request to update `OWNERS_ALIASES` file when need to update members.
- Add `OWNERS` files to children folders by teams if they need.

**Comment by wuhuizuo** [comment]
> > Do I have permission to review the code of these modules? If I don't have it, why take away my permission?
> 
> It should be like the current `sessionctx` and `config` folders. The merge permission is restricted. The review(+1) permission is the same.

Yes, only changed on approving for these folders.

File: bindinfo/OWNERS
Comment (wuhuizuo) [inline bindinfo/OWNERS:3]: Just for `approvers` here since the field is already set up.
if `reviewers` field is setup, the reviewers will also not inherit from parent owners.

> We hacked the prow instance to support this feature.

File: OWNERS_ALIASES
Comment (fixdb) [reply OWNERS_ALIASES:28]: I agree with @hawkingrei

File: OWNERS_ALIASES
Comment (wuhuizuo) [inline OWNERS_ALIASES:18]:
```suggestion
    - fixdb
    - winoros
    - qw4990
```

File: OWNERS_ALIASES
Comment (wuhuizuo) [inline OWNERS_ALIASES:23]:
```suggestion
    - fixdb
    - winoros
    - qw4990
```

File: OWNERS_ALIASES
Comment (wuhuizuo) [inline OWNERS_ALIASES:28]:
```suggestion
    - fixdb
    - winoros
    - qw4990
```

**Comment by wuhuizuo** [comment]
@chrysan I have removed you from the members, PTAL.

File: OWNERS_ALIASES
Comment (wuhuizuo) [reply OWNERS_ALIASES]: Fixed

**Comment by hi-rustin** [comment]
I don't think this is a scale way to do it. So I propose we close it. I also don't see procedural justice here, do you mean other approvers/committers from the community are not valid approvers/committers?

**Comment by wuhuizuo** [comment]
> I don't think this is a scale way to do it. So I propose we close it. I also don't see procedural justice here, do you mean other approvers/committers from the community are not valid approvers/committers?

Subset of the repository level approvers are the approvers of these paths.

File: OWNERS_ALIASES
Comment (wuhuizuo) [inline OWNERS_ALIASES:127]:
```suggestion
    - Arenatlx
    - elsa0520
    - fixdb
    - hawkingrei
    - hi-rustin
    - qw4990
    - time-and-fate
    - winoros
```

**Comment by wuhuizuo** [comment]
/hold

wait for all members of `sig-approvers-planner` become committees.

File: OWNERS_ALIASES
Comment (wuhuizuo) [inline OWNERS_ALIASES:125]:
```suggestion
    - AilinKid
```

**Review by fixdb — APPROVED** [review]
+1

**Pull request merged**
